generator/component/temp.e2e.js

```diff
@@ -0,0 +1,16 @@
+describe('<%= upCaseName %> Route', () => {
+
+	// Before each test
+	beforeEach(() => {
+
+		// Navigate to <%= name %> component
+    browser.get('http://localhost:3000/<%= name %>');
```

Comment:
urls called with browser get are relative to the base (you dont need to put localhost:3000 there)